sdks/python/apache_beam/io/gcp/datastore/v1/datastoreio.py

```diff
@@ -313,8 +313,11 @@ class _Mutate(PTransform):
   supported, as the commits are retried when failures occur.
   """
 
-  # Max allowed Datastore write batch size.
+  # Max allowed Datastore writes per batch, and max bytes per batch.
+  # Note that the max bytes per batch is lower than the actual limit enforced
```

Comment:
Is not 5MB to restrictive, if 10MB is the limit? Would not this create unnecessary RPCs?